src/pages/contact/index.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
```

Comment:
This is _close_ to what we want. We dropped the `ListArrow`s [in master](https://github.com/COVID19Tracking/website/blob/master/src/pages/contact/index.js). Please update this file to reflect that.

Reply:
@schwartzadev ah, gotcha. See 1ce523d.